extensions/widgets/header/header.lcb

```diff
@@ -249,56 +281,142 @@ Parameters:
+Syntax: get the colorScheme of <widget>
 
-Summary: Whether the background of the header bar is drawn opaque or not
+Summary: The color scheme of the widget (Android only)
+
+Parameters:
+pColorScheme(enum): The color scheme of the widget
+-"Light"
+-"Dark"
+-"Red"
+-"Pink"
+-"Purple"
+-"Deep Purple"
+-"Indigo"
+-"Blue"
+-"Light Blue"
+-"Cyan"
+-"Teal"
+-"Green"
+-"Light Green"
+-"Lime"
+-"Yellow"
+-"Amber"
+-"Orange"
+-"Deep Orange"
+-"Brown"
+-"Grey"
+-"Blue Grey"
+
+Description:
+Use the <colorScheme> property to set the color scheme of the widget, Android only.
+*/
+property colorScheme get mColorScheme set setColorScheme
+metadata colorScheme.editor is "com.livecode.pi.enum"
+metadata colorScheme.options is "Light,Dark,Red,Pink,Purple,Deep Purple,Indigo,Blue,Light Blue,Cyan,Teal,Green,Light Green,Lime,Yellow,Amber,Orange,Deep Orange,Brown,Grey,Blue Grey"
+metadata colorScheme.default is "Teal"
+metadata colorScheme.label is "Color Scheme"
+
+/*
+Syntax: set the leftAction of <widget> to <pIconName>
+Syntax: get the leftAction of <widget>
+
+Summary: The action on the left of the header
+
+Parameters:
+pIconName(enum): The name of the icon of the left action of the header
+- a list of predefined values
 
 Description:
-Use the <opaqueBackground> property to determine whether objects underneath the header bar are visible or not.
+Use the <leftAction> property to set the icon of the widget's left action to one of the predefined values.
+The complete list of preset values is available by executing
 
+``` put iconNames()```
+
+Android only.
 */
-property opaqueBackground get mOpaque set setOpaque
-metadata opaqueBackground.default is "true"
-metadata opaqueBackground.label is "Opaque Background"
+property leftAction get mLeftAction set setLeftAction
+metadata leftAction.editor is "com.livecode.pi.enum"
+metadata leftAction.options is "execute: get iconNames()"
+metadata leftAction.default is "reorder"
+metadata leftAction.label is "Left Action"
 
 /*
-Syntax: set the <showDivide> of <widget> to {true | false}
-Syntax: get the <showDivide> of <widget>
+Syntax: set the titleVisibility of <widget> to {true | false}
+Syntax: get the titleVisibility of <widget>
 
-Summary: Whether the dividing line at the top of the header bar is drawn or not
+Summary: Whether the title is visible or not
+
+Description:
+Use this property to determine whether the title is visible (true) or not (false), Android only.
+*/
+property titleVisibility get mTitleIsVisible set setTitleVisibility
+metadata titleVisibility.default is "true"
+metadata titleVisibility.label is "Title Visibile"
+
+/*
+Syntax: set the distinctTitle of <widget> to {true | false}
+Syntax: get the distinctTitle of <widget>
+
+Summary: Whether the title is distinct or not
 
 Description:
-Use the <showDivide> property to determine whether the dividing line at the top of the header bar is drawn or not.
+Use this property to determine whether the title distinct (true) or not (false), Android only.
```

Comment:
It would be useful to have a little explanation of what the effect of having distinctTitle true or false is here.